### ISSUE TITLE
Route cache not working

### ISSUE THREAD

**Issue by niddu85** [dulnan/nuxt-multi-cache#17]
After creating a blank nuxt project with
`npx nuxi@latest init my-app`

and enabling route caching on redis, it doesn't work

`package.json`

```
{
  "name": "nuxt-app",
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@types/node": "^18",
    "nuxt": "^3.4.3"
  },
  "dependencies": {
    "nuxt-multi-cache": "^3.0.0"
  }
}
```

`nuxt.config.ts`

```
export default defineNuxtConfig({
  modules: ["nuxt-multi-cache"],
  multiCache: {
    route: {
      enabled: true,
    },
  },
});
```

`app\multiCache.serverOptions.ts`

```
import redisDriver from "unstorage/drivers/redis";
import { defineMultiCacheOptions } from "nuxt-multi-cache";

export default defineMultiCacheOptions({
  route: {
    storage: {
      driver: redisDriver({
        db: 4,
        host: "127.0.0.1",
        port: 6379,
        base: "route:",
      }),
    },
  },
});
```

`app.vue`

```
<script setup lang="ts">
useRouteCache((helper) => {
  helper.setCacheable().setMaxAge(9000);
});
</script>

<template>
  <div>
    <NuxtWelcome />
  </div>
</template>
```

**Comment by dulnan** [comment]
Thank you very much for raising this issue. Thanks to this I found a rather [embarrassing typo](https://github.com/dulnan/nuxt-multi-cache/commit/445aa5b8356aad62563ed954cca8ab115890b7cc#diff-030fc083b2cbf5cf008cfc0c49bb4f1b8d97ac07f93a291d068d81b4d1416f70L80) that broke the route cache in the 3.0.0 release. I have fixed it now and released 3.0.1. I was successfully able to setup route caching using redis.
